packages/web-react/src/components/Header/README.md

```diff
@@ -4,398 +4,541 @@ The Header is a highly variable and customizable component. It comes in several
-| `children`         | `ReactNode`                | —       | ✕        | Children node                          |
-| `color`            | [`primary` \| `secondary`] | —       | ✕        | HeaderDesktopActions background color  |
-| `UNSAFE_className` | `string`                   | —       | ✕        | HeaderDesktopActions custom class name |
-| `UNSAFE_style`     | `CSSProperties`            | —       | ✕        | HeaderDesktopActions custom style      |
+Toggle button is already part of the mobile actions component. It is linked to
+the [Header Dialog](#header-dialog) via the `dialogId` prop.
 
-The rest of the properties are based on the type of `HTMLElement`. [Docs][HTMLElementDocs]
+#### Custom Mobile Actions
 
-## HeaderDialog
+You can place any custom content into the mobile actions component:
 
-### API
+```jsx
+<HeaderMobileActions dialogId="my-header-dialog">{/* Mobile-only actions */}</HeaderMobileActions>
+```
+
+#### API
 
 | Name               | Type                                           | Default | Required | Description                     |
 | ------------------ | ---------------------------------------------- | ------- | -------- | ------------------------------- |
 | `children`         | `ReactNode`                                    | —       | ✕        | Children node                   |
-| `id`               | `string`                                       | —       | ✔        | Dialog ID                       |
-| `isOpen`           | `bool`                                         | `false` | ✔        | Open state                      |
-| `onClose`          | `(event: ClickEvent or KeyboardEvent) => void` | —       | ✔        | Callback for dialog when closed |
-| `UNSAFE_className` | `string`                                       | —       | ✕        | HeaderDialog custom class name  |
-| `UNSAFE_style`     | `CSSProperties`                                | —       | ✕        | HeaderDialog custom style       |
+| `dialogId`         | `string`                                       | —       | ✔        | ID of the linked HeaderDialog   |
+| `isOpen`           | `bool`                                         | `false` | ✔        | Dialog open state               |
+| `menuToggleLabel`  | `string`                                       | `Menu`  | ✕        | Label of the menu toggle button |
+| `onOpen`           | `(event: ClickEvent or KeyboardEvent) => void` | —       | ✔        | Callback for dialog when opened |
+| `UNSAFE_className` | `string`                                       | —       | ✕        | Custom class name               |
+| `UNSAFE_style`     | `CSSProperties`                                | —       | ✕        | Custom style                    |
 
-The rest of the properties are based on the type of `<dialog>`. [Docs][DialogElementDocs]
+The component implements the [`HTMLElement`][mdn-api-html-element] interface.
```

Comment:
This is here from the original README. I'm wondering we might better link to the actual element docs (`<header>`, `<div>`, `<span>`).

Reply:
We agreed we can improve it next time.